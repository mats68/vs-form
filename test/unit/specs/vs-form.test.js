import Vue from 'vue'
import { shallow } from 'vue-test-utils'

import VsForm from 'src/components/vs-form'
import VsItem from 'src/components/vs-item'

import { miniSchema } from 'vs-schema'

import Vuetify from 'vuetify'
Vue.use(Vuetify)

// todo Vue.use
Vue.component('vs-form', VsForm)
Vue.component('vs-item', VsItem)

describe('VsForm miniSchema', () => {
  let Cmp

  beforeEach(() => {
    Cmp = shallow(VsForm, {
      propsData: {
        schema: miniSchema
      }
    })
  })

  it('has the expected html structure', () => {
    expect(Cmp.element).toMatchSnapshot()
  })
})
