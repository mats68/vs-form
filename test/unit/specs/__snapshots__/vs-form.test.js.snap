// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VsForm miniSchema has the expected html structure 1`] = `<!---->`;
